source/basic.tex

```diff
@@ -1367,8 +1367,8 @@
 The declarations found by name lookup shall either all denote the same entity or
 shall all denote functions or function templates;
 in the latter case,
-the declarations are said to form a set of overloaded
-functions\iref{over.load}. Overload resolution\iref{over.match}
+the declarations are said to form an \defn{overload set}.
```

Comment:
I think this definition is not completely clear: if name lookup finds exactly one function, is that an overload set or not? (Presumably these days the answer should be yes.) How about something more like:

> If the declarations found by name lookup all denote functions or function templates, the declarations are said to form an *overload set*. The declarations found by name lookup shall either all denote the same entity or form an overload set.

Reply:
Sounds good. Fixed.